ui/src/app/components/app-router.tsx

```diff
@@ -20,15 +28,18 @@ export const usePath = () => {
   return path;
 };
 
-const useQuery = () => {
-  const searchString = useLocation().search.replace(/^\?/, '');
+export function parseQueryParams(queryParams: string) {
+  const searchString = queryParams.replace(/^\?/, '');
   return querystring.parse(searchString);
+}
+
+export const useQuery = () => {
+  return parseQueryParams(useLocation().search);
```

Comment:
There is a function here that should probably be reconciled with this:

https://github.com/all-of-us/workbench/blob/master/ui/src/app/utils/navigation.tsx#L95-L104

That function I think is problematic because it uses a hook in a utility, but the thing I like about it is that usage of `URLSearchParams`, which I think may be a better interface.

Reply:
I'll switch to using `URLSearchParams` in `parseQueryParams` and then use that for `useQuery` as before.

Reply:
Using a hook inline like this scares me; I believe when we enable the hook linter eventually, it will not like this. I would rather assign the value on a separate line to avoid tempting fate. There's enough magic around hooks that I could imagine this causing an obscure issue.